src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -33,12 +33,6 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <WarningsAsErrors />
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='DebugPlaceholder|AnyCPU'">
```

Comment:
Moved this to c`ommon.props` so it's available in all projects which imports that.